File: HFSGuardaDiretorio_GtkSharp_C#/gui/FrmSobre.cs
using System;

namespace HFSGuardaDiretorio.gui
{
	public partial class FrmSobre : Gtk.Dialog
	{
		public FrmSobre ()
		{
			this.Build ();
		}

		protected void OnButtonOkClicked (object sender, EventArgs e)
		{
			this.Hide();
		}
	}
}

File: HFSGuardaDiretorio_GtkRuby/gui/FrmSobre.rb
require "mscorlib"
require "System, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"

module HFSGuardaDiretorio.gui
	class FrmSobre < Gtk::Dialog
		def initialize()
			self.Build()
		end

		def OnButtonOkClicked(sender, e)
			self.Hide()
		end
	end
end